src/js/cookieMessage.js

```diff
@@ -2,6 +2,45 @@ class CookieMessage {
 
 	constructor (CookieMessageEl) {
 		this.CookieMessageEl = CookieMessageEl;
+
+		return this.setup();
+	}
+
+	setup () {
+		const message = document.querySelector('.o-cookie-message');
```

Comment:
for js, we use data attributes to hook to the DOM. in this case, CookieMessageEl in the constructor should be that. You can take a look at what the init function does, and here's another example: https://github.com/Financial-Times/o-header/blob/master/src/js/header.js#L10